magento/magento2
# #274 Admin Global Search: Search categories

### Description (*)
Global search in backend can search into products, orders, customers, pages and config sections.
Now we can search categories too.

### Fixed Issues (if relevant)
1. Fixes https://github.com/magento/community-features/issues/274

### Manual testing scenarios (*)
1. Login into backend
2. Create a test category "Try to find me"
3. Use global search (in header to right, next to admin account setting)
4. Enter "to find"
5. You will see in autocomplete list our new fresh category, link is valid and redirect to the edit form

### Contribution checklist (*)
